server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/version/v66/DbVersion66.java

```diff
@@ -37,6 +37,10 @@ public void addSteps(MigrationStepRegistry registry) {
       .add(1807, "Add on project_branches key", AddIndexOnProjectBranchesKey.class)
       .add(1808, "Add branch column to projects table", AddBranchColumnToProjectsTable.class)
       .add(1809, "Populate project_branches with existing main branches", PopulateMainProjectBranches.class)
+      .add(1810, "Add ANALYSIS_UUID in WEBHOOK_DELIVERIES", AddAnalysisUuidToWebhookDeliveries.class)
+      .add(1811, "Make column CE_TASK_UUID nullable", UpdateCeTaskUuidColumnToNullableOnWebhookDeliveries.class)
+      .add(1812, "Populate ANALYSIS_UUID of WEBHOOK_DELIVERIES", PopulateAnalysisUuidColumnOnWebhookDeliveries.class)
+      .add(1813, "Make ANALYSIS_UUID not null in WEBHOOK_DELIVERIES", UpdateCeTaskUuidColumnToNullableOnWebhookDeliveries.class)
```

Comment:
this migration becomes useless since analysis_uuid can be null

Reply:
:warning: :warning: the name of migration 1813 is the same as 1811 :warning: :warning: